src/utils/installPackage.js

```diff
@@ -1,8 +1,21 @@
 const spawn = require('cross-spawn');
 const config = require('./config');
 const path = require('path');
+const resolve = require('resolve');
 
-module.exports = async function(dir, name) {
+const installPeerDependencies = async (dir, name) => {
+  let basedir = path.dirname(dir);
+
+  const resolved = resolve.sync(name, {basedir});
+  const pkg = await config.load(resolved, ['package.json']);
+  const peers = pkg.peerDependencies || {};
+
+  for (const peer in peers) {
+    await install(dir, `${peer}@${peers[peer]}`, false);
```

Comment:
I think you can pass multiple packages to install at once to npm or yarn rather than doing each one in series. Does that not work?

Reply:
Also, do we also need to install peers of peers etc? Not sure.

Reply:
> I think you can pass multiple packages to install at once to npm or yarn rather than doing each one in series. Does that not work?

This should work, just didn't think about it at the time. 

> Also, do we also need to install peers of peers etc? Not sure.

I'm explicitly blocking it right now to prevent following a theoretically endless dependency chain. I doubt we'd see this "in the wild" though. Consider the following:

- Package A lists Package B as a peer dep
- Package B lists Package C as a peer dep

If Package B _requires_ Package C to run, then the author of Package A is essentially guaranteeing their module won't work by just installing it's own listed dependencies. This seems unlikely...